### PR TITLE
Move VAOS redux code into feature folders

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This moves the redux code into feature folders, according to our agreed upon layout. I've also moved the flow files into the associated feature folders.

## Testing done
Unit testing

## Acceptance criteria
- [ ] Organization matches our agreed standard

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/tests/list/cc-appointments.unit.spec.jsx [hunk 1]
```diff
@@ -45,7 +44,6 @@ describe('VAOS integration: upcoming CC appointments', () => {
       queryByText,
     } = renderWithStoreAndRouter(<FutureAppointmentsList />, {
       initialState,
-      reducers,
```
Comment (jbalboni) [inline src/applications/vaos/tests/list/cc-appointments.unit.spec.jsx:48]: Our reducers are all included by the helper we're using, so this isn't necessary

File: src/applications/vaos/containers/ClinicChoicePage.jsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ import {
   updateFormData,
   routeToNextAppointmentPage,
   routeToPreviousAppointmentPage,
-} from '../actions/newAppointment.js';
+} from '../new-appointment/redux/actions.js';
```
Comment (narin) [inline src/applications/vaos/containers/ClinicChoicePage.jsx:16]: This was present already, but we don't need the `.js` here.

File: src/applications/vaos/containers/CommunityCarePreferencesPage.jsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ import {
   updateFormData,
   routeToNextAppointmentPage,
   routeToPreviousAppointmentPage,
-} from '../actions/newAppointment.js';
+} from '../new-appointment/redux/actions.js';
```
Comment (narin) [inline src/applications/vaos/containers/CommunityCarePreferencesPage.jsx:16]: Same as above

File: src/applications/vaos/containers/ContactInfoPage.jsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ import {
   updateFormData,
   routeToNextAppointmentPage,
   routeToPreviousAppointmentPage,
-} from '../actions/newAppointment.js';
+} from '../new-appointment/redux/actions.js';
```
Comment (narin) [inline src/applications/vaos/containers/ContactInfoPage.jsx:13]: Same as above

File: src/applications/vaos/containers/DateTimeRequestPage.jsx [hunk 5]
```diff
@@ -6,7 +6,7 @@ import {
   onCalendarChange,
   routeToNextAppointmentPage,
   routeToPreviousAppointmentPage,
-} from '../actions/newAppointment.js';
+} from '../new-appointment/redux/actions.js';
```
Comment (narin) [inline src/applications/vaos/containers/DateTimeRequestPage.jsx:9]: Same as above

File: src/applications/vaos/containers/DateTimeSelectPage.jsx [hunk 6]
```diff
@@ -10,7 +10,7 @@ import {
   routeToPreviousAppointmentPage,
   startRequestAppointmentFlow,
   requestAppointmentDateChoice,
-} from '../actions/newAppointment.js';
+} from '../new-appointment/redux/actions.js';
```
Comment (narin) [inline src/applications/vaos/containers/DateTimeSelectPage.jsx:13]: Same as above

**Review by narin — APPROVED** [review]
LGTM.  Just a minor comment on some unneeded `.js` extension on some imports but those were already present in the code.

**Pull request merged**